features/version.feature

```diff
@@ -13,7 +13,7 @@ Feature: ec cli version subcommand
     OPA                         v.+
     Conftest                    v.+
     Red Hat AppStudio \(shared\)  v.+
-    Cosign                      v.+
+    Cosign                      (v.+|N\/A)
```

Comment:
This (and below) is odd, we don't get a version when using snapshot dependencies